src/manifest.json

```diff
@@ -11,6 +11,7 @@
 	"content_scripts": [
 		{
 			"matches": ["https://*.librarything.com/*"],
+			"all_frames": true,
```

Comment:
what dis

Reply:
UM ok so the main reason is, the search results in LibraryThing are in an iframe. This just makes our stuff apply to iframes as well.